File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task167_CLAUDE_claude-3-5-sonnet-20240620.java

public class CubeDiagonal {
    public static double cubeDiagonal(double volume) {
        double side = Math.cbrt(volume);
        double diagonal = Math.sqrt(3) * side;
        return Math.round(diagonal * 100.0) / 100.0;
    }

    public static void main(String[] args) {
        System.out.println(cubeDiagonal(8));
        System.out.println(cubeDiagonal(343));
        System.out.println(cubeDiagonal(1157.625));
    }
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task167_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>
#include <math.h>

double cubeDiagonal(double volume) {
    double side = cbrt(volume);
    double diagonal = sqrt(3) * side;
    return round(diagonal * 100.0) / 100.0;
}

int main() {
    printf("%.2f\
", cubeDiagonal(8));
    printf("%.2f\
", cubeDiagonal(343));
    printf("%.2f\
", cubeDiagonal(1157.625));
    return 0;
}
